IFC4x3/Properties/w/WindDirectionRange_1DH_IenR1EWgR3G1pApMRJ/Documentation.md

```diff
@@ -0,0 +1 @@
+Indicate the range of wind direction the anemometrograph can monitor.
```

Comment:
Generalize?

Reply:
Generalized the description and changed name to WindAngleRange as @SergejMuhic suggested: https://github.com/bSI-InfraRoom/IFC-Specification/pull/165/commits/1e8b0740cc2e7e000d58ef2f240b7e5a49640ee9